hoomd/state.py

```diff
@@ -122,7 +122,7 @@ class State:
     - :math:`d`: diameter :math:`[\\mathrm{length}]` - deprecated in v3.0.0.
       HOOMD-blue reads and writes particle diameters, but does not use them in
       any computations. As of the current beta release, diameter is still used
-      in `md.pair.SLJ` and `md.pair.DLVO`.
+      in `md.pair.DLVO`.
```

Comment:
Remove the "As of the current beta release" sentence. DLVO no longer uses diameters.